tests/test_operator.py

```diff
@@ -7,6 +7,7 @@
                     SparseFunction, SparseTimeFunction, Dimension, error, SpaceDimension,
                     NODE, CELL, dimensions, configuration, TensorFunction,
                     TensorTimeFunction, VectorFunction, VectorTimeFunction, switchconfig)
+from devito import  Le, Lt, Ge, Gt  # noqa
```

Comment:
should be merged with the line above? the noqa is because they're used in `str` and then `eval` ?

Reply:
`#` noqa is because flake cannot 'see' them later in tests as when passed as arguments. Also I understand that `#`noqa cannot be applied to multi-line parenthese-import wrapping.

Reply:
it can, needs to be at the end of the first line, anyway np here